packages/terra-date-picker/src/DatePickerField.jsx

```diff
@@ -222,13 +229,25 @@ const DatePickerField = (props) => {
     mergedInputAttrs = { 'aria-describedby': ariaDescriptionIds, ...inputAttributes };
   }
 
+  const helpLabel = help ? (
+    <div id="format" aria-label={`Date Format: ${intl.formatMessage({ id: 'Terra.datePicker.dateFormat' })}, ${help}`}>
```

Comment:
Should this `id` also be randomized like the other places?

Reply:
The field component doesn't require a description id as they internally provide `helpDescriptionIds`. I had format just to give that div an id.